v3/integration/config.go

```diff
@@ -105,10 +106,44 @@ func loadConfig(file string) (*config, error) {
 	if err := json.Unmarshal(jsonBytes, &c); err != nil {
 		return nil, err
 	}
-
+	problems := findProblemsInTheConfig(jsonBytes, &c)
+	if len(problems) != 0 {
+		return nil, errors.New(strings.Join(problems, "\n"))
+	}
 	return &c, nil
 }
 
+// findProblemsInTheConfig tries keep the configuration honest with regard
+// to aspects such as duplicate entries with in the Expected field.
+func findProblemsInTheConfig(configBytes []byte, c *config) []string {
```

Comment:
Do you have thoughts about other config problems you may try to find with this function in the future?  Otherwise, findDuplicateConfigEntries might be a more precise name for the function.

Reply:
Yeah, I was thinking of other things like not being able to declare a result set for a lint that doesn't exist (which also happened in #613 )

Reply:
Fair enough, the name seems reasonable then.